tt_metal/impl/device/device.cpp

```diff
@@ -3980,35 +1867,6 @@ std::vector<CoreCoord> Device::get_optimal_dram_bank_to_logical_worker_assignmen
     return this->optimal_dram_bank_to_logical_worker_assignment_;
 }
 
-HalProgrammableCoreType Device::get_programmable_core_type(CoreCoord virtual_core) const {
```

Comment:
This PR undoes (some of? all?) changes @ayerofieiev-tt made in https://github.com/tenstorrent/tt-metal/pull/16259 -- is this intentional, or bad rebase?

Reply:
Bad rebase I think, let me check on this

Reply:
Should be fixed now